gtk4/Gir.toml

```diff
@@ -1834,6 +1834,8 @@ generate_builder = false
 [[object]]
 name = "Gtk.Snapshot"
 status = "generate"
+final_type = false
```

Comment:
This one is actually a final type or not?

Reply:
It is https://docs.gtk.org/gtk4/class.Snapshot.html, but it is a workaround to get the functions of a GtkSnapshot generated in a SnapshotExt trait. Will document that

Reply:
Why do you need them in a `SnapshotExt` trait though? You can only call the `gtk_snapshot*` functions on `GtkSnapshot`, not on `GdkSnapshot`.